### PR TITLE
Add awslimitchecker

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

_Created with Grayskull_

### CODE REVIEW

File: recipes/awslimitchecker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7610ac0fbe0817754e238099053a4ff4d3ea5260676094af86331891d6934a09
+
+build:
+  entry_points:
```
Comment (BastianZim) [inline recipes/awslimitchecker/meta.yaml:14]: Is `entry_points` correctly defined? Grayskull wasn't able to extract those.

Reference: https://github.com/jantman/awslimitchecker/blob/645e22dbe98841421c095837b621eb1188b732a2/setup.py#L81-L84

**Comment by BastianZim** [comment]
@conda-forge/help-python Would be ready for review, thanks!

File: recipes/awslimitchecker/meta.yaml (hunk 1 above)
Comment (mariusvniekerk) [reply recipes/awslimitchecker/meta.yaml:14]: Yeah those are usually defined as a dictionary, probably why grayskull couldn't extract

**Review by mariusvniekerk — APPROVED** [review]

File: recipes/awslimitchecker/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/awslimitchecker/meta.yaml:14]: Ahh ok, thanks!

**Comment by BastianZim** [comment]
Thank you!

**Pull request merged**